### DISCUSSION TITLE
Undervolting ryzen ai 9 hx 370 cpu?

### DISCUSSION THREAD

**Discussion by zocsen** [seerge/g-helper#2942] [General]
Hi! I was wondering if it's possible to undervolt these ryzen hx 370 cpu-s and if yes, is this feature coming for the new g16 variants?

**Comment by seerge** [comment]
@zocsen hello, i don’t think it’s possible

**Reply by Bondorudo1** [reply]
Is it still not possible? I have g16 with this cpu and there is an option to undervolt it. I am not an expert though and not sure if it actually does anything.

**Reply by seerge** [reply]
@Bondorudo1 hello, it turned out to be possible, and since v.0.186 you can do that under `Fans+Power` -> `Advanced`. Even tho effect is minimal, same as for any other AMD CPU :)

**Reply by Bondorudo1** [reply]
Thanks! Good to know. I was wondering if I waste my time toggling it in different modes. Also, is there a way to add my mouse to g helper? It doesn’t show up, maybe it is not supported: ASUS MD200 SMARTO

**Reply by seerge** [reply]
@Bondorudo1 it's up to you with undervolting. As for the mouse - it's not ROG or TUF, so it has never been in Armoury Crate neither, right ? What exactly do you want to configure in your mouse and how do you do this right now ?

**Reply by Bondorudo1** [reply]
I did have it in Armory Crate. It appeared there and I could update drivers from there for it and all the staff available with it.

**Reply by seerge** [reply]
@Bondorudo1 if you did have it in AC, then you can [fill in template to request it's support](https://github.com/seerge/g-helper/issues/new?assignees=&labels=&projects=&template=peripherals_request.yml) and include all data asked there :)

**Reply by Bondorudo1** [reply]
Thank you! I didn’t take screenshots, but I was able to find online images. Not sure if they are from the latest update or not. I hope it’s not an issue.

**Reply by Bondorudo1** [reply]
Hello again! I noticed an issue, but I am not sure if it’s a problem on g helper side or not. When I limit batter to 80% or lower, then I get this weird constant sound from the middle of my laptop, is this considered normal? When I change it back to 100%, problem disappears. Has anybody mention this before or maybe found a solution? Laptop zephyrus g16 2024 with amd ai

**Reply by seerge** [reply]
@Bondorudo1 that most likely just a coil whine :) it has nothing to do with apps. It affects lots of electronic devices, just google it.

**Reply by Bondorudo1** [reply]
I tried to turn back on ASUs services and limit from myAsus, sound disappeared, but it could be due to restarting the laptop. I will observe and see how it goes
